src/Faithlife.Utility/StringUtility.cs

```diff
@@ -78,6 +78,38 @@ public static string ReplaceOrdinal(this string source, string oldValue, string
 #endif
 		}
 
+#if NETSTANDARD2_0
+		/// <summary>
+		/// Calls string.Contains with StringComparison.Ordinal.
+		/// </summary>
+#else
+		/// <summary>
+		/// Calls <see cref="string.Contains(char, StringComparison)">string.Contains</see> with <see cref="StringComparison.Ordinal">StringComparison.Ordinal</see>.
+		/// </summary>
+#endif
+		public static bool ContainsOrdinal(this string source, char value) =>
+#if NETSTANDARD2_0
+			source.Contains(value);
+#else
+			source.Contains(value, StringComparison.Ordinal);
+#endif
+
+#if NETSTANDARD2_0
+		/// <summary>
+		/// Calls string.Contains with StringComparison.Ordinal.
+		/// </summary>
+#else
+		/// <summary>
+		/// Calls <see cref="string.Contains(char, StringComparison)">string.Contains</see> with <see cref="StringComparison.Ordinal">StringComparison.Ordinal</see>.
+		/// </summary>
+#endif
+		public static bool ContainsOrdinal(this string source, string value) =>
```

Comment:
Unnecessary duplicate of `string.Contains`, which has always performed an ordinal comparison; delete.

https://docs.microsoft.com/en-us/dotnet/api/system.string.contains?view=netframework-4.8#System_String_Contains_System_String_